cmd/app/options/audit.go

```diff
@@ -23,3 +24,11 @@ func (a *AuditOptions) AddFlags(fs *pflag.FlagSet) *AuditOptions {
 	a.AuditOptions.AddFlags(fs)
 	return a
 }
+
+func (a *AuditOptions) DynamicConfigurationFlagChanged(cmd *cobra.Command) bool {
+	if ff := cmd.Flag("audit-dynamic-configuration"); ff != nil && ff.Changed {
+		return true
```

Comment:
It seems like we want to know if the flag is true, not if it has *changed*? Given the default is presumable `false`, this does work, but it seems a bit confusing.. any reason for the `changed` approach?

Reply:
Yes, even setting it to "false" means your trying to interact with that feature in some way. That's the way I look at it anyway. Want to make sure it's explicit that this is not supported.

Happy to change it to `== true`

Reply:
🤷‍♂ it feels weird, if the option is there, to exit if I've set it to the same value as the default... I'd expect that to work as a user personally. Otherwise can you remove the option from the FlagSet altogether otherwise?

Reply:
I wasn't able to find a way no, I'll change it to that